src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -616,6 +617,9 @@ static uct_ud_iface_ops_t uct_ud_mlx5_iface_ops = {
     .iface_is_reachable       = uct_ib_iface_is_reachable,
     .iface_query              = uct_ud_mlx5_iface_query,
 
+    .iface_progress_enable    = uct_ud_iface_progress_enable,
```

Comment:
Let's add UD implementation in another PR, and keep this PR to add UCT API only.
We want to remote wakeup FD support from UD altogether, and rely on wakeup callback.